### PR TITLE
Add manual conversation reload command /reload_now

### PR DESCRIPTION
**Pull request by crdev**
Reloads conversation_tree.textproto from GitHub (main branch) and re-creates the updater.
The command is enabled for admins only.
Drive-by: Increase the count of top nodes reported to 20

### CODE REVIEW

File: stats.py [hunk 1]
```diff
@@ -14,10 +14,13 @@
 class Stats:
     timestamp_by_user: Dict[str, float]
     interactions: Counter
+    last_reload_time_tz: datetime.datetime
 
     def __init__(self):
         self.timestamp_by_user = dict()
         self.interactions = Counter()
+        self.last_reload_time_tz = None
+        self.reloader_username = None
```
Comment (edganiukov) [inline stats.py:23]: I guess it is not mandatory, but this field is not defined as a class field

File: stats.py [hunk 2]
```diff
@@ -40,9 +47,19 @@ def compute(self) -> str:
...
         uptime = now_tz - STARTTIME_TZ
         uptime = datetime.timedelta(seconds=int(uptime.total_seconds()))
-        return f"Start time: {STARTTIME_TZ.strftime(DATETIME_FORMAT)}\nUptime: {uptime}\nTotal users:\n{user_stats}\nTop 10 interactions:\n{node_stats}"
+        output = [
+            f"Start time: {STARTTIME_TZ.strftime(DATETIME_FORMAT)}",
+            f"Uptime: {uptime}"
+        ]
+        if self.last_reload_time_tz:
+            output.append(f"Last conversation reload: {self.last_reload_time_tz.strftime(DATETIME_FORMAT)} ({self.reloader_username})")
+        output += [
```
Comment (edganiukov) [inline stats.py:61]: you can use `output.extend` instead of `+`

**Review by edganiukov — CHANGES_REQUESTED** [review]

File: stats.py (hunk 1 above)
Comment (crdev) [reply stats.py:23]: Done.

File: bot.py [hunk 3]
```diff
@@ -411,18 +444,53 @@ def create_keyboard_options(node_by_name):
...
+    if username in ADMIN_USERS:
+        logger.info(f"Reloading conversation from {CONVERSATION_TREE_URL}")
+        convo_buffer = None
+        try:
+            with urllib.request.urlopen(CONVERSATION_TREE_URL) as f:
+                convo_buffer = f.read().decode("utf-8")
+            reset_bot_data(convo_buffer, update)
+            bot_stats.conversation_reloaded(username)
+            logger.info(f"Reload successful, killing updater")
+            os.kill(os.getpid(), RELOAD_SIGNAL)
```
Comment (skd) [inline bot.py:458]: Why do we need to restart the bot?

File: stats.py (hunk 2 above)
Comment (edganiukov) [reply stats.py]: imo, `extend` is more obvious what is happening there, but no strong opinion there

**Review by edganiukov — APPROVED** [review]

File: bot.py (hunk 3 above)
Comment (crdev) [reply bot.py]: Made sure it is not necessary. Reverted.

File: stats.py (hunk 2 above)
Comment (crdev) [reply stats.py]: Thanks, fixed!

**Review by skd — APPROVED** [review]

**Pull request merged**